app/controllers/questions/questions_controller.rb

```diff
@@ -33,7 +33,7 @@ def update
     end
 
     def current_path(params = {})
-      question_path(self.class.to_param, params)
+      request.path
```

Comment:
AL and i took an idle friday afternoon look at this and are wondering, do we even need `current_path` at all? it seems to mostly be used in `form_with`s in `erb` files to post to ones own path which A) might be the default anyway and B) if not, `request` can be accessed in the view so maybe doesn't need this helper. a little late in the day to be sure but leaving you with this thought in case it's useful.

Reply:
@wschaefer What if we remove this hunk from the diff? Once we add the W-2s-specific code, we don't need this change I think.